documentation/IDTA-01002-3/modules/ROOT/pages/specification/interfaces-payload.adoc

```diff
@@ -132,29 +132,46 @@ h|*Interface* h|*interface-shortName*
 |Asset Administration Shell Basic Discovery Interface |AAS-DISCOVERY
 |===
 
-The value for the interface attribute shall be “\{interface-shortName}-\{interface-version}”.
+Which can be combined with the following serialisation modifiers:
+[%autowidth,width="100%",options="header",]
+|===
+h|*Serialisation Modifier* h|*serialisation-modifier-value*
+|Metadata | METADATA
+|ValueOnly | VALUE
```

Comment:
Why not with Path and Reference? Did we decide to not yet include them?

Reply:
Didn't really saw a need yet. Have added them.